bin/ldnode.js

```diff
@@ -97,6 +97,11 @@ var argv = require('nomnom')
     full: 'default-app',
     help: 'URI to use as a default app for resources (default: https://linkeddata.github.io/warp/#/list/)'
   })
+  .option('signup', {
```

Comment:
This is actually just to sign up the first user.
Single user mode right now is done by not setting the flag --idp

This is like --create-admin



On Fri, Jan 22, 2016 at 6:00 AM -0800, "Dmitri Zagidulin" <notifications@github.com> wrote:












In bin/ldnode.js:
> @@ -97,6 +97,11 @@ var argv = require('nomnom')
>      full: 'default-app',
>      help: 'URI to use as a default app for resources (default: https://linkeddata.github.io/warp/#/list/)'
>    })
> +  .option('signup', {



Can we rename this option to singleUserMode (from our in-person discussion / #201?)



—
Reply to this email directly or view it on GitHub.